apps/auction2/stylesheets/index.styl

```diff
@@ -19,7 +19,23 @@
   clearfix()
 
 .auction2-commercial-filter
+  margin-top 20px
   border-top 1px solid gray-lighter-color
 
-  .cf-sidebar__filter.cf-sidebar__price
+  .cf-sidebar-title__mediums, .cf-sidebar-title__artists
+    padding-top 20px
+    margin-top 20px
+    border-top 1px solid gray-lighter-color
+    padding-bottom 0px
+
```

Comment:
Just FYI, little BEM hack that @acjay made us aware of.

```stylus
.cf-sidebar
  &__filter, &__followed_artists
    // styles
  &__aggregated_artists
    // styles
```

This knowledge might be outdated soon but just in case its useful :)